src/database.ts

```diff
@@ -1744,13 +1974,16 @@ class Database extends ServiceObject {
    * region_tag:spanner_create_storing_index
    * Creating a storing index:
    */
-  updateSchema(statements, callback) {
+  updateSchema(
+    statements: Schema,
+    callback: UpdateSchemaCallback
+  ): Promise<[database_admin_client.longrunning.Operations, GaxOperation]> {
     if (!is.object(statements)) {
       statements = {
         statements: arrify(statements),
-      };
+      } as Schema;
```

Comment:
tried with `if (typeof statements !== 'object')` but it failed some test cases other things changed as per suggession.

Reply:
Right, because `Array` is typeof 'object'

you can then do
```ts
if(typeof statements === 'string' || Array.isArray(statements)) {
      statements = {
        statements: arrify(statements),
      };
    }
```